src/index.ts

```diff
@@ -237,6 +237,23 @@ class Envoy {
       const hha_hash = url.searchParams.get('hha_hash');
       log.normal("%s (%s) connection for HHA ID: %s", anonymous ? "Anonymous" : "Agent", agent_id, hha_hash);
 
+      // Signal is a message initiated in conductor which is sent to UI. In case to be able to route signals
```

Comment:
```suggestion
      // Signal is a message initiated in conductor which is sent to UI. To be able to route signals
```

Reply:
:P I didn't write that comment but that is an improvement.